src/auth/auth.ts

```diff
@@ -46,6 +64,10 @@ export class BBAuth {
     BBAuth.tokenCache = {};
   }
 
+  private static build1BBUrl(scs: string, service: string, zone: string, pathAndQuery: string) {
```

Comment:
I don't see where this is being called anywhere.  Was it a victim of refactoring?

Reply:
Victim of refactoring.  I'll kill this, if we're ready to go forward w/ unit tests.